app/scripts/controllers/form.js

```diff
@@ -35,7 +38,10 @@ angular.module('ludwig').controller('FormCtrl', function($scope, $http, $state,
         $scope.submitting = true;
         var test = _.pick($scope.test, ['_id', 'situation', 'name', 'description', 'expectedResults', 'keywords']);
         $http.put(config.baseApiPath + '/acceptance-tests/' + test._id, test).then(function() {
-            $state.go('index.show', { 'testId': test._id });
+            AcceptanceTestsService.launchTest(test).
```

Comment:
Always put the `.` with the following property (i.e. break line _before_ it, not after).